docs/getting-started.md

```
@@ -0,0 +1,101 @@
+# شروع کار با Cursor AI
+
+## نصب و راه‌اندازی
+
+### سیستم‌های مورد پشتیبانی
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)